l10n_ar_edi_ux/models/account_move.py

```diff
@@ -88,3 +89,26 @@ def _post(self, soft=True):
             raise UserError(msg)
 
         return super(AccountMove, self - purchase_ar_edi_invoices)._post(soft=soft)
+
+    def _get_permissions(self):
+        """ Get 'permiso de embarque' for foreign invoices. """
+        self.ensure_one()
+        res = []
+        for permiso in self.l10n_ar_boarding_permission_ids:
+            res.append({'Id_permiso': permiso.l10n_ar_boarding_permission_number, 'Dst_merc': permiso.l10n_ar_boarding_permission_dst_country.l10n_ar_afip_code})
+        return res
+
+    @api.model
+    def wsfex_get_cae_request(self, last_id, client):
+        """ Set permiso de embarque to foreign invoice. """
+        res = super(AccountMove, self).wsfex_get_cae_request(last_id, client)
+        ArrayOfPermisions = client.get_type('ns0:ArrayOfPermiso')
+        if int(self.l10n_latam_document_type_id.code) == 19 and int(self.l10n_ar_afip_concept) == 1:
+            permisos = self._get_permissions()
+            permiso_existente = "S" if permisos else "N"
+        else:
+            permisos = []
+            permiso_existente = ""
+        res.update({'Permisos': ArrayOfPermisions(permisos) if permisos else None})
+        res.update({'Permiso_existente': permiso_existente})
```

Comment:
not need the else clause.
The original method already has this resolved.

Only the If is enough to add the permissions info if exist and needed 

```suggestion
        if int(self.l10n_latam_document_type_id.code) == 19 and int(self.l10n_ar_afip_concept) == 1:
            permisos = self._get_permissions()
            res.update({'Permisos': ArrayOfPermisions(permisos) if permisos else None})
            res.update({'Permiso_existente': "S" if permisos else "N"})
```